dev/RatingControl/RatingControl.cpp

```diff
@@ -46,6 +47,29 @@ float RatingControl::ActualRatingFontSize()
     return RenderingRatingFontSize() / 2;
 }
 
+inline double RatingControl::ItemSpacing()
+{
+    // stars are rendered 2x size and we use expression animation to shrink them down to desired size, which will create those spacings (not system margin).
```

Comment:
Comment lines are long, making it hard to review on GitHub without having to use the scrollbar. Keep to 100 columns.